io/zenoh-transport/src/unicast/transport_unicast_inner.rs

```diff
@@ -80,7 +80,6 @@ pub(crate) trait TransportUnicastTrait: Send + Sync {
     /*************************************/
     /*            TERMINATION            */
     /*************************************/
-    async fn close_link(&self, link: Link, reason: u8) -> ZResult<()>;
```

Comment:
I don't know the Transport layer enough to tell if this should be removed or not. Maybe add a note to ask Luca when he comes back ?

Reply:
@OlivierHecart @fuzzypixelz I've checked the code, it seems that this method is not needed, we can remove it